lib/cartopy/mpl/geoaxes.py

```diff
@@ -1691,11 +1712,7 @@ def _pcolormesh_patched(self, *args, **kwargs):
 
```

Comment:
You need to update the comment above here. Your changes have modified `to_mask` to be shape `(Ny-1, Nx-1)`

Reply:
done